terraform/aws/scripts/install.sh.tpl

```diff
@@ -6,7 +6,13 @@ sudo apt-get update -y
 sudo apt-get install -y curl unzip
 
 # Download Vault into some temporary directory
-curl -L "${download_url}" > /tmp/vault.zip
+vault_download_url=https://releases.hashicorp.com/vault/${download_version}/vault_${download_version}_linux_amd64.zip
+
+if [[ ! -z "${download_url}" ]]; then
```

Comment:
I find positive conditions easier to read than negative ones.  I might suggest changing the `! -z` to `-n`, as I believe they are equivalent.  I don't _think_ the double brackets are required here either.  I think single brackets would suffice.